athena/simulate_pipeline.sh

```
@@ -0,0 +1,52 @@
+# Run tests for each module using its own virtual environment
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)